Core/DISET/private/BaseClient.py

```diff
@@ -205,7 +205,7 @@ def __findServiceURL( self ):
       
     if len( urls ) == len( self.__bannedUrls ):
       self.__bannedUrls = []  # retry all urls
-      self.__retryDelay = 5 / len ( urls )  # we run only one service! In that case we increase the retry delay.
+      self.__retryDelay = 5 / len ( urls ) if len( urls ) > 1 else 1  # we run only one service! In that case we increase the retry delay.
```

Comment:
Okay, I increased to 3. In that case we will have 5 retry and each connection will be 3 retry. In that case we will wait 13 second.
But when ipv6 enabled we wait 5 * 3 * (3*0.5) = 22.5